src/EventTickets/Migrations/CreateEventsTable.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Give\EventTickets\Migrations;
+
+use Give\Framework\Database\DB;
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Framework\Migrations\Exceptions\DatabaseMigrationException;
+
+/**
+ * @unreleased
+ */
+class CreateEventsTable extends Migration {
+    /**
+     * @inheritdoc
+     */
+    public static function id() {
+        return 'give-events-create-events-table';
+    }
+
+    public static function title() {
+        return 'Create give_events table';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public static function timestamp() {
+        return strtotime( '2022-01-29 00:00:00' );
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function run() {
+        global $wpdb;
+
+        $table   = $wpdb->give_events;
+        $charset = DB::get_charset_collate();
+
+        $sql = "CREATE TABLE $table (
+			id INT UNSIGNED NOT NULL,
+			form_id INT NULL,
+			description TEXT NULL,
+			start_datetime DATETIME NULL,
+			end_datetime DATETIME NULL,
+			ticket_close_datetime DATETIME NULL,
```

Comment:
What's the purpose of this column? I couldn't find any mention to it.

Reply:
> Last date & time tickets will be available

We need a way to (optionally) disable ticket sales independently from the event date.

For example, "Tickets are available up until one week before the event."

Reply:
In that sense, we could also have a start date: `sales_start_datetime` and `sales_end_datetime`. But, once this is not part of the current scope and could easily be added later, I'd be inclided to drop it for now and implement that column we really decide to have that feature. What do you think?

Reply:
It is low cost to leave the column here as compared to introducing an additional database migration to add it. Since the "spec" for this feature is unclear let's just leave it and move on.